gradle/test-intellij.versions.toml

```diff
@@ -1,3 +1,3 @@
 [versions]
 # https://jb.gg/intellij-platform-builds-list
-versionList = "IIC-243.21565.193,IIC-223.8836.41"
+versionList = "IIC-243.23654.189"
```

Comment:
Should this be 251 something?

Reply:
This file is no longer used after migration to the 2.x intellij plugin. Removed.